sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/metrics/_exporter.py

```diff
@@ -174,6 +185,64 @@ def _convert_point_to_envelope(
     return envelope
 
 
+# pylint: disable=protected-access
+def _handle_std_metric_envelope(
+    envelope: TelemetryItem,
+    name: str,
+    attributes:Mapping[str, AttributeValue]
+) -> Optional[TelemetryItem]:
+    properties = {}
+    tags = envelope.tags
+    # TODO: switch to semconv constants
+    status_code = attributes.get("http.status_code", None)
+    if name == "http.client.duration":
+        properties["_MS.MetricId"] = "dependencies/duration"
+        properties["_MS.IsAutocollected"] = "True"
+        properties["Dependency.Type"] = "http"
+        properties["Dependency.Success"] = str(_is_status_code_success(status_code, 400))
+        target = None
+        if "peer.service" in attributes:
+            target = attributes["peer.service"]
+        elif "net.peer.name" in attributes:
+            if attributes["net.peer.name"] is None:
+                target = None
+            elif "net.host.port" in attributes and \
+                attributes["net.host.port"] is not None:
+                target = "{}:{}".format(
+                    attributes["net.peer.name"],
+                    attributes["net.host.port"],
+                )
+            else:
+                target = attributes["net.peer.name"]
+        properties["dependency/target"] = target
+        properties["dependency/resultCode"] = str(status_code)
+        # TODO: operation/synthetic
+        properties["cloud/roleInstance"] = tags["ai.cloud.roleInstance"]
+        properties["cloud/roleName"] = tags["ai.cloud.role"]
+    elif name == "http.server.duration":
+        properties["_MS.MetricId"] = "requests/duration"
+        properties["_MS.IsAutocollected"] = "True"
+        properties["request/resultCode"] = str(status_code)
+        # TODO: operation/synthetic
+        properties["cloud/roleInstance"] = tags["ai.cloud.roleInstance"]
+        properties["cloud/roleName"] = tags["ai.cloud.role"]
+        properties["Request.Success"] = str(_is_status_code_success(status_code, 500))
+    else:
+        # Any other autocollected metrics are not supported yet for standard metrics
+        # We ignore these envelopes in these cases
+        return None
```

Comment:
Related to my comment in the trace exporter. Since there are other autocollected metrics we are not supporting yet, shouldn't we leave those associated telemetry w/o ProcessedByMetricExtractors.

Reply:
See my previous comment.